Samples/V1.0Samples/LocalMediaSamples/PolicyRecordingBot/ServiceConfiguration.Local.cscfg

```diff
@@ -3,25 +3,38 @@
   <Role name="CRWorkerRole">
     <Instances count="1" />
     <ConfigurationSettings>
-      <Setting name="Microsoft.WindowsAzure.Plugins.Diagnostics.ConnectionString" value="DefaultEndpointsProtocol=https;AccountName=$storage$;AccountKey=$storageKey$" />
       <Setting name="ServiceDnsName" value="%ServiceDns%" />
-      <!-- xyz.cloudapp.net-->
-      <Setting name="ServiceCNAME" value="%CName%" />
-      <!-- CNAME pointing to the .cloudapp.net if available.-->
+		<!-- xyz.cloudapp.net-->
+		<!-- Service DNS Name: 
+         This setting defines the DNS name for the service. 
+         It should be the fully qualified domain name (FQDN) used to access the service, like 'myservice.cloudapp.net'. 
+         Replace %ServiceDns% with your service's DNS name. -->
+		<Setting name="ServiceCNAME" value="%CName%" />
+		<!-- CNAME pointing to the .cloudapp.net if available.-->
+		<!-- Service CName: 
+         This setting is commonly used for creating a CNAME record that points to the service's DNS name.
+         It allows you to map a custom domain name to the service’s DNS name. 
+         Replace %ServiceCName% with the desired CNAME, such as 'custom.mydomain.com'. -->
       <Setting name="DefaultCertificate" value="ABC0000000000000000000000000000000000CBA" />
       <Setting name="APPINSIGHTS_INSTRUMENTATIONKEY" value="" />
     </ConfigurationSettings>
     <Certificates>
       <Certificate name="Default" thumbprint="ABC0000000000000000000000000000000000CBA" thumbprintAlgorithm="sha1" />
     </Certificates>
   </Role>
-  <NetworkConfiguration>
-    <AddressAssignments>
-      <InstanceAddress roleName="CRWorkerRole">
-        <PublicIPs>
-          <PublicIP name="instancePublicIP" domainNameLabel="pip" />
-        </PublicIPs>
-      </InstanceAddress>
-    </AddressAssignments>
-  </NetworkConfiguration>
+	<NetworkConfiguration>
+		<!-- Name of VirtualNetworSite.-->
+		<VirtualNetworkSite name="PolicyRecordingBot" />
```

Comment:
Fix indentation for this section to be like the above section, use spaces not tabs.

Reply:
done